util/devel/test/portability/vagrant/current/ubuntu-jammy64-homebrew/Vagrantfile

```diff
@@ -5,6 +5,16 @@
 # configures the configuration version (we support older styles for
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
+
+# TODO: Migrate this configuration over to Apptainer. Michael ran into initial
+# difficulties with this because:
+# - Homebrew wants to install parts of itself in system directories outside of
+#   /home, and Apptainer wants those to be bind mounted from the host,
```

Comment:
IMO it wouldn't matter if it wanted to install into /home. On ARM macs, it wants to install into /opt/homebrew, so you could mention that here.